### DISCUSSION TITLE
Datable.js

### DISCUSSION THREAD

**Discussion by dridk** [thegeeklab/hugo-geekdoc#715] [Polls]
Hi,

I made shortcodes to display data in [Datatable.js](https://datatables.net)  .

I can make a pull request for geekdoc if you are agree.
It works as follow as inline or block:

```
# inline
{{< datatable url='data.json' >}}

 Or 

# block
{{< datatable-content >}} 

{   [ .... ] } 

{{</ datatable-content>}}

```

**Comment by xoxys** [comment]
Hi, thanks that would be cool, but I would prefer to add the `datatable` version only that supports local files as well.
